src/sql/workbench/contrib/notebook/browser/notebookStyles.ts

```diff
@@ -250,7 +250,7 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean, conf
 			collector.addRule(`.notebook-preview .rangeHighlight { background-color: ${notebookFindMatchHighlightColor};}`);
 		}
 		if (notebookFindRangeHighlightColor) {
-			collector.addRule(`.notebook-preview .rangeSpecificHighlight { background-color: ${notebookFindRangeHighlightColor}!important;}`);
```

Comment:
Why are you changing this? And what about the notebookFindMatchHighlightColor above?

Reply:
Previously we only looked at rendered markdown which is under .notebook-preview class but now to extend it to outputs which does not fall under that, I removed it and made more generic.